jpegenc/models/rgb2ycbcr/commons.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
```

Comment:
Hi Merk,

we should have three testing flavors for `testbench`; native, VHDL and Verilog simulation. You could keep the relevant part in separate files, or work with a single version, but this would require to pass the options down to a refined `testbench(target_mode)`